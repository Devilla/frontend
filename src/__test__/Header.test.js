// import React from 'react';
// import { shallow, mount } from 'enzyme';
// import renderer from 'react-test-renderer'
// import configureStore from 'redux-mock-store'
// import { Provider } from 'react-redux'
// import {createStore} from 'redux'
// import { load, loaded } from 'ducks/loading';
// import { Header } from 'components';
// import { store } from 'App';
//
// // Snapshot for Home React Component
// describe('>>> Header --- Snapshot',()=>{
//     it('+++capturing Snapshot of Header', () => {
//         const renderedValue =  renderer.create(<Header store={store} />).toJSON()
//         expect(renderedValue).toMatchSnapshot();
//     });
// });
