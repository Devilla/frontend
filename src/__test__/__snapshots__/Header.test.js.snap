// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Header --- Snapshot +++capturing Snapshot of Header 1`] = `
<div
  className="customer-header"
>
  <div
    className="progress"
  >
    <div
      aria-valuemax={100}
      aria-valuemin={0}
      aria-valuenow={0}
      className="progress-bar progress-bar-info"
      role="progressbar"
      style={
        Object {
          "width": "0%",
        }
      }
    />
  </div>
  <div />
  <div
    className="nav-topbar-flex"
  >
    <div
      className="topbar-left"
    >
      <h4>
        <a
          onClick={[Function]}
          style={Object {}}
        >
          <i
            className="icon-arrow-left mr-3"
          />
        </a>
      </h4>
    </div>
    <ul
      className="list-unstyled topbar-right-menu float-right mb-0 nav-custom-header  rounded-circle"
    >
      <li
        className="dropdown notification-list"
      >
        <a
          aria-expanded="false"
          aria-haspopup="false"
          className="nav-link dropdown-toggle nav-user"
          data-toggle="dropdown"
          role="button"
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          <i
            className=" mdi mdi-account-settings-variant"
          />
          <span
            className="ml-1"
          />
        </a>
        <div
          className="dropdown-menu dropdown-menu-right dropdown-menu-animated profile-dropdown"
        >
          <div
            className="dropdown-item noti-title"
          >
            <h6
              className="text-overflow m-0"
            />
          </div>
          <a
            className="dropdown-item notify-item"
            href="javascript:void(0);"
          >
            <i
              className="fi-head"
            />
            <span>
              Your Profile
            </span>
          </a>
          <a
            className="dropdown-item notify-item"
            href="javascript:void(0);"
          >
            <i
              className="fi-help"
            />
            <span>
              Support
            </span>
          </a>
          <a
            className="dropdown-item notify-item"
            href="javascript:void(0);"
          >
            <i
              className="fi-power"
            />
            <span>
              Logout
            </span>
          </a>
        </div>
      </li>
    </ul>
  </div>
</div>
`;
